posthog/api/routing.py

```diff
@@ -43,8 +46,19 @@ class StructuredViewSetMixin(_GenericViewSet):
         JwtAuthentication,
         PersonalAPIKeyAuthentication,
         authentication.SessionAuthentication,
-        authentication.BasicAuthentication,
     ]
+    additional_authentication_classes = []
```

Comment:
Idea: Why not turn it around?

If the user in the view specifies `permission_classes`, these get added. While here we have `base_permission_classes` under the hood.

Reply:
Yeah I actually just thought about that as well :D

Reply:
I think its annoying magic tbh. The code is already a bit hard to follow and moving even more stuff around security to some default value that is hard to trace feels error prone.

Also the new logic I changed, wouldn't support that anways.